lib/web_ui/test/canvaskit/embedded_views_test.dart

```diff
@@ -554,6 +554,26 @@ void testMain() {
       result = diffViewList(<int>[3, 4], <int>[1, 2, 3, 4, 5, 6]);
       expect(result, isNull);
     });
+
+    test('does not crash when a prerolled platform view is not composited',
+        () async {
+      ui.platformViewRegistry.registerViewFactory(
+        'test-platform-view',
+        (int viewId) => html.DivElement()..id = 'view-0',
+      );
+      await _createPlatformView(0, 'test-platform-view');
+
+      final EnginePlatformDispatcher dispatcher =
+          ui.window.platformDispatcher as EnginePlatformDispatcher;
+
+      LayerSceneBuilder sb = LayerSceneBuilder();
+      sb.pushOffset(0, 0);
+      sb.pushClipRect(ui.Rect.zero);
+      sb.addPlatformView(0, width: 10, height: 10);
+      sb.pop();
```

Comment:
Checking the DOM for the absence of `<flt-platform-view-slot>` might help avoid false positives (although the check itself must not be prone to false negatives :thinking:).

Reply:
Done